src/rez/utils/pip.py

```diff
@@ -383,7 +399,8 @@ def get_rez_requirements(installed_dist, python_version, name_casings=None):
 
     # assume package is platform- and arch- specific if it isn't pure python
     is_pure_python = is_pure_python_package(installed_dist)
-    if not is_pure_python:
+    has_entry_points_scripts = is_entry_points_scripts_package(installed_dist)
```

Comment:
Could you add a little comment to explain why it's required? Someone unfamiliar with Windows might not know that scripts in Windows are executables.